### PR TITLE
Cli tab: added Save button

### PR DESCRIPTION
**Pull request by limonspb**
By @ctzsnooze  request adding "Save" button. Technically its a new feature, so need to think about adding to 10.8. But it's a very small change.

The "Cancel" button is not needed cuz (by @KarateBrot) users can just click disconnect or to any other tab.
Renamed Save/Load to File to Import/Export from/to file

image

### CODE REVIEW

File: locales/en/messages.json
Comment (limonspb) [reply locales/en/messages.json]: fixed

**Review by KarateBrot — DISMISSED** [review]

File: locales/en/messages.json
Comment (McGiverGim) [inline locales/en/messages.json:2878]: Please, add a "description" field to all the messages. This description is used by the translators to get more information about the string. Something like:
 ```suggestion
         "message": "Export to file",
         "description": "Option in the CLI tab to export the contents to a file"
```

**Review by McGiverGim — COMMENTED** [review]
I don't like the idea of a save button, but I like the renaming to "import/export" to let them clear that it does not save the changes.

**Review by chmelevskij — APPROVED** [review]

File: locales/en/messages.json
Comment (limonspb) [reply locales/en/messages.json]: done!

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by ctzsnooze** [comment]
The text doesn't need to be so verbose.  

|  Current  |  Suggested  |  Comment
|:-------------------:|:-------------:|:--------------:|
|  Copy to Clipboard  |  Copy  |  Where else could we copy too?  |
|  Clear output history  |  Clear  |  What else would we clear?  |
|  Export to File  |  Export  |  What else would we export too?  |
|  Import from File  |  Import  |  What else would we import from?  |

**Comment by McGiverGim** [comment]
If we are going to start again to change texts, please stop this until the new version of the Configurator. Any change of text invalidate the translations, and all the translators must translate it again.

**Comment by limonspb** [comment]
> If we are going to start again to change texts, please stop this until the new version of the Configurator. Any change of text invalidate the translations, and all the translators must translate it again.

good call. I will revert the text changes.

**Review by KarateBrot — APPROVED** [review]

**Comment by ctzsnooze** [comment]
This isn't a bug fix for 4.3, it was just something that I know people found a bit confusing.

How about we try to make the changes 'really good', but mark them for the 4.4 / 10.9 milestone, not 4.3?

**Comment by limonspb** [comment]
> This isn't a bug fix for 4.3, it was just something that I know people found a bit confusing.
> 
> How about we try to make the changes 'really good', but mark them for the 4.4 / 10.9 milestone, not 4.3?

yeah, whatever works :)

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by limonspb** [comment]
don't see a need, cleaning PRs

**Pull request closed without merging**
